### ISSUE TITLE
DTMF events

### ISSUE THREAD

**Issue by benlangfeld** [rayo/xmpp#28]
The events should come from an input component (as with other signal detectors). Should one be started at the top of the call?

**Comment by benlangfeld** [comment]
Solution: Input component w/ hotword detection (SRGS `GARBAGE`?) and repeat-on-match (send a match event and reset the buffer), started prior to answering incoming calls.

This means we need to specify repeat-on-match.

**Comment by crienzo** [comment]
Would this be a new kind of intermediate event where the input component continues to execute?

**Comment by benlangfeld** [comment]
Yes it would, but there's still no consensus on wether it should be grammar
based or something else. This needs work.

Regards,
Ben Langfeld

Em 27 Mar 2013, às 23:00, Chris Rienzo notifications@github.com escreveu:

Would this be a new kind of intermediate event where the input component
continues to execute?

—
Reply to this email directly or view it on
GitHubhttps://github.com/rayo/xmpp/issues/28#issuecomment-15564318
.
